### DISCUSSION TITLE
Minimun Hardware requirements

### DISCUSSION THREAD

**Discussion by cristian98149** [kairos-io/kairos#2245] [Q&A]
Hi All,

I was wondering about the minimum hardware requirements for running just kairos with the default kubernetes apps.

On getting-started page I don't see them:
![image](https://github.com/kairos-io/kairos/assets/47886306/31d50d2e-8b06-4678-9680-4539b4051fbb)

Thank you

BR

**Comment by jimmykarily** [comment]
We could do some testing to see how many resources Kairos needs in various scenarios (e.g. with/without p2p, with/without k3s, etc) but I would take this with a (big) grain of salt because Kairos is not the real user's workload, so one would have to add the workload on top.

Calculating how many resources will be needed on top for some workload (e.g. an application running on Kubernetes) might be trickier than it sounds because it's not always Kairos + application resources but it's also all the overhead to get the application running on Kubernetes (e.g. controllers reconciling things in the background etc).

Having said that, such "minimum requirements" would only serve as an absolute minimum below which one should not even try, rather than a recommendation for hardware requirements. As long as we are clear about it, I think it would be nice to have those in the docs.

@kairos-io/maintainers what do you think?
